src/Promenade.Web/Views/Home/Locations.cshtml

```diff
@@ -0,0 +1,160 @@
+﻿@model Ocuda.Promenade.Models.Entities.Location
+@{
+    ViewData["Title"] = "Locations";
+}
+    <body>
+        <h1>Find A Library</h1>
+        <h2>Find your closest Maricopa County Library by entering your address or zip code:</h2>
```

Comment:
Need to remove "Maricopa County Library" from here and "Library" from the line above.